src/components/Leaderboard/Leaderboard.module.css

```diff
@@ -1,34 +1,54 @@
 .leaderboard {
```

Comment:
Next person who get leaderBoard is to remove this CSS module.

Reply:
yeah. this is a temporary change. it will be changed soon